api/framer-motion.api.md

```diff
@@ -21,15 +21,19 @@ import { RefObject } from 'react';
 import { SpringProps } from 'popmotion';
 import { SVGAttributes } from 'react';
 
+// Warning: (ae-forgotten-export) The symbol "AnimatePresenceProps" needs to be exported by the entry point index.d.ts

```

Comment:
This warning should also be avoided

Reply:
These are both quite weird. `AnimatePresenceProps_2` isn't even defined and I do export `AnimatePresenceProps` from the root file.

Reply:
Deleted `types` and re-ran, is all good now